angular/projects/spark-angular/src/lib/components/sprk-dropdown/sprk-dropdown.component.ts

```diff
@@ -18,11 +18,11 @@ import {
       <div [ngClass]="{ 'sprk-o-Box': dropdownType === 'mastheadSelector' }">
         <sprk-link
           [additionalClasses]="getTriggerClasses()"
-          ariaHasPopUp="true"
           (click)="toggle($event)"
           [idString]="idString"
           [analyticsString]="analyticsString"
-          role="combobox"
+          aria-haspopup="listbox"
+          [attr.aria-label]="title ? triggerText : screenReaderText"
```

Comment:
if triggerText or screenReaderText aren't supplied, we should have a reasonable default